src/main/java/seedu/address/storage/JsonAdaptedRoom.java

```diff
@@ -4,24 +4,33 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.patient.Patient;
 import seedu.address.model.room.Room;
 
 public class JsonAdaptedRoom {
 
+    public static final String PATIENT_PRESENT_IS_OCCUPIED_FALSE = "When patient is present isOccupied cannot be false";
+
     private int roomNumber;
     private boolean isOccupied;
     //TODO
-    private String patient;
+    //private String patient;
```

Comment:
do you want to consider removing this comment? is it needed?

Reply:
Okay